src/android/Capture.java

```diff
@@ -537,6 +554,51 @@ private Uri whichContentStore() {
         }
     }
 
+    @Nullable
+    private Bitmap rotateAccordingToExifOrientation(Uri uri) {
+        Context context = this.cordova.getContext();
+        int orientation;
+        Bitmap bitmap;
+        try {
+            InputStream inputStream = context.getContentResolver().openInputStream(uri);
+            if (inputStream == null)
+                throw new IOException("input stream from ContentResolver is null");
+            ExifInterface ei = new ExifInterface(inputStream);
+            orientation = ei.getAttributeInt(ExifInterface.TAG_ORIENTATION, ExifInterface.ORIENTATION_UNDEFINED);
+            bitmap = MediaStore.Images.Media.getBitmap(context.getContentResolver(), uri);
+        } catch (IOException e) {
+            LOG.e(LOG_TAG, "Failed reading bitmap", e);
+            return null;
+        }
+
+        Bitmap rotatedBitmap;
+        switch (orientation) {
+            case ExifInterface.ORIENTATION_ROTATE_90:
+                rotatedBitmap = rotateImage(bitmap, 90);
+                break;
+
+            case ExifInterface.ORIENTATION_ROTATE_180:
+                rotatedBitmap = rotateImage(bitmap, 180);
+                break;
+
+            case ExifInterface.ORIENTATION_ROTATE_270:
+                rotatedBitmap = rotateImage(bitmap, 270);
+                break;
+
+            case ExifInterface.ORIENTATION_NORMAL:
+            default:
+                rotatedBitmap = bitmap;
+        }
+        return rotatedBitmap;
+    }
+
+    private Uri fromBitmapToUri(Context context, Bitmap inImage) {
+        ByteArrayOutputStream bytes = new ByteArrayOutputStream();
+        inImage.compress(Bitmap.CompressFormat.JPEG, 85, bytes);
```

Comment:
hardcoded quality? Is this smart?

Reply:
As far as I know there's no param in this plugin to set a specific image quality for the standard image capture. If we add a param to set quality it should work for standard images and rotated images, but this was not related to the issue. Anyway I agree, it's not smart, what would you suggest?

Reply:
yes

Reply:
Why can't we just set it to `100`?

Reply:
That's an option, sure

Reply:
85 seems a bit random. 100 is still hard-coded, but at least it won't interfere with the quality wanted - they can always change the quality later.

Is that what we'll go with then?

Reply:
Ok, the compression is now 100 abed3674d13a5f380c5c1f779ae49752f0d5c9f1